docs/user-guide/projects.md

```diff
@@ -315,9 +315,10 @@ stringData:
 ```
 
 !!! warning
-Please keep in mind when using a project-scoped repository, only applications from the same project can make use of
-it. When using applicationsets with the Git generator, only non-scoped repositories can be used (i.e. repositories that
-do _not_ have a `project` set).
+Please keep in mind when using a project-scoped repository, only applications or applicationsets with a matching project 
```

Comment:
Can we document this somewhere in the applicationSet section of the doc as well? :)